multicluster/test/integration/suite_test.go

```diff
@@ -105,7 +102,13 @@ var _ = BeforeSuite(func() {
 	}
 
 	var err error
-	cfg, err = testEnv.Start()
+	done := make(chan interface{})
+	go func() {
+		defer GinkgoRecover()
+		cfg, err = testEnv.Start()
+		close(done)
+	}()
+	Eventually(done).WithTimeout(1 * time.Minute).Should(BeClosed())
```

Comment:
Is this based on some official Ginkgo examples?

Reply:
yes, kind of? I found it in one of their issue, not in official documentation. I put the link of the issue in the PR description. Here is the exact example given by Ginkgo: https://github.com/onsi/ginkgo/issues/882#issuecomment-1014693378